src/assets/levels/PlaydateProjectTest.ldtk

```diff
@@ -5198,6 +5198,372 @@
 				}
 			],
 			"__neighbours": [ { "levelIid": "9682b430-25d0-11ef-ba44-e352beaf71c7", "dir": "o" }, { "levelIid": "445f5ef0-25d0-11ef-ba44-7b5ec6701e46", "dir": "o" } ]
+		},
```

Comment:
just for info: feel free to expand on this "physics test" test level to have multiple levels side-by-side to run consecutive tests. Think like a test lab :)